tests/v2/integration/steveapi/steve_api_test.go

```diff
@@ -1598,6 +1632,34 @@ func formatJSON(obj *clientv1.SteveCollection) ([]byte, error) {
 			delete(data[i].(map[string]interface{})["metadata"].(map[string]interface{}), "managedFields")
 			delete(data[i].(map[string]interface{})["metadata"].(map[string]interface{}), "uid")
 			data[i].(map[string]interface{})["metadata"].(map[string]interface{})["resourceVersion"] = "1000"
+			// replace url in links
+			links := data[i].(map[string]interface{})["links"].(map[string]interface{})
+			// replace remove
+			removeLink := links["remove"]
+			if removeLink != nil {
+				replaceURL = urlRegex.ReplaceAllString(removeLink.(string), defautlUrlString)
+				links["remove"] = replaceURL
+			}
+			// replace self
+			selfLink := links["self"]
+			if selfLink != nil {
+				replaceURL = urlRegex.ReplaceAllString(selfLink.(string), defautlUrlString)
+				links["self"] = replaceURL
+			}
+			// replace update
+			updateLink := links["update"]
+			if updateLink != nil {
+				replaceURL = urlRegex.ReplaceAllString(updateLink.(string), defautlUrlString)
+				links["update"] = replaceURL
+			}
+			// replace view
+			replaceLink := links["view"]
+			if replaceLink != nil {
+				replaceURL = urlRegex.ReplaceAllString(replaceLink.(string), defautlUrlString)
+				links["view"] = replaceURL
+			}
+			data[i].(map[string]interface{})["links"] = links
+
```

Comment:
We can simplify all this - on line 1673 (past the end of this diff) do

```
jsonString = urlRegex.ReplaceAllString(jsonString, defautlUrlString)
```

Then all the string replaces in individual parts of the map can be removed.

Reply:
I completely miss this, made it much simpler, thank you!